tool/tctl/common/token_command.go

```diff
@@ -39,8 +41,8 @@ import (
 	"github.com/gravitational/trace"
 )
 
-// TokenCommand implements `tctl token` group of commands
-type TokenCommand struct {
+// TokensCommand implements `tctl tokens` group of commands
+type TokensCommand struct {
```

Comment:
Since you renamed it here, you probably will have to make corresponding change to enterprise code as well:

https://github.com/gravitational/teleport.e/blob/master/tool/tctl/main.go#L19

TBH I would just keep the old name `TokenCommand` to save the trouble.

Reply:
I don't mind making the change as it's so small, I'd prefer for the name of the struct to be consistent with the command name. I'll make a PR in enterprise after this gets merged.

Reply:
@capnspacehook In that case, enterprise changes should be made simultaneously with the open-source. Otherwise, anyone who tries to update enterprise after this merges won't be able to compile it with the latest master.

Reply:
Ok, opened a PR here: https://github.com/gravitational/teleport.e/pull/433

Reply:
Sounds good, don't forget to bump the e submodule reference in this repo after the enterprise PR merges so it points to the correct commit in teleport.e.